website/content/middleware/timeout.md

```diff
@@ -58,6 +72,64 @@ TimeoutErrorHandlerWithContext func(error, echo.Context) error
 DefaultTimeoutConfig = TimeoutConfig{
     Skipper:      DefaultSkipper,
     Timeout:      0,
-    ErrorHandler: nil,
+    ErrorMessage: "",
+}
+```
+
+## Alternatively handle timeouts in handlers
+
+```go
+func main() {
+    e := echo.New()
+
+    doBusinessLogic := func(ctx context.Context, UID string) error {
+        // NB: Do not use echo.JSON() or any other method that writes data/headers to client here. This function is executed
+        // in different coroutine that should not access echo.Context and response writer
+
+        log.Printf("uid: %v\n", UID)
+        //res, err := slowDatabaseCon.ExecContext(ctx, query, args)
```

Comment:
Please remove this comment

Reply:
I'll add note here. Using methods with contexts has maybe sideeffects that are not wanted - i.e. when timeout we set to request reached context will be cancelled and therefore methods with contexts could prematurely returns